run/README.md

```diff
@@ -16,6 +16,7 @@
 |[Cloud SQL (MySQL)][mysql]               | Use MySQL with Cloud Run | - |
 |[Cloud SQL (Postgres)][postgres]         | Use Postgres with Cloud Run | - |
 |[Hello Broken][hello_broken]             | Something is wrong, how do you fix it? | [<img src="https://storage.googleapis.com/cloudrun/button.svg" alt="Run on Google Cloud" height="30"/>][run_button_hello_broken] |
+|[Events – Pub/Sub][events_pubsub]         | Use Pub/Sub with Cloud Run | - |
```

Comment:
I think we should have this description parallel the Pub/Sub description more, and be moved right under it. Might want to flag it as alpha if we're going to "advertise" it here.

```suggestion
|[Events – Pub/Sub][events_pubsub]         | Event-driven service with Events for Cloud Run for Pub/Sub | - |
```

Reply:
I agree we can move the Pub/Sub trigger next to the other Pub/Sub sample.

I disagree that we should change the title in this way.
Cloud Run is not_for_ Pub/Sub.
Pub/Sub is a trigger/event type for Cloud Run.

Once public, we should remove the non-native Pub/Sub sample.

Reply:
The core feedback here is that the two samples should be listed next to each other, and with similar enough structure in their descriptions to make differentiation easy.

Whether one of them is removed at GA is a good question.